### DISCUSSION TITLE
luci显示异常

### DISCUSSION THREAD

**Discussion by lexin8** [immortalwrt/immortalwrt#1488] [Q&A]
### Describe the bug

新编译固件不显示中文
.config
`CONFIG_TARGET_x86=y
CONFIG_TARGET_x86_64=y
CONFIG_TARGET_KERNEL_PARTSIZE=60
CONFIG_TARGET_ROOTFS_PARTSIZE=600
![88](https://github.com/user-attachments/assets/2c5795d0-f655-41de-acee-bd105977522d)

### ImmortalWrt version

immortalwrt最新的master分支

### ImmortalWrt release

immortalwrt最新的master分支

### ImmortalWrt target/subtarget

immortalwrt最新的master分支

### Device

x86

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by libiunc** [comment]
不选chn导致的
[参考](https://github.com/libiunc/user-FAQ/blob/main/%E5%B0%8F%E7%99%BD%E9%97%AE%E9%A2%98%E9%80%9F%E6%9F%A5.md#%E8%87%AA%E7%BC%96%E8%AF%91%E7%9A%84%E5%9B%BA%E4%BB%B6%E6%97%A0%E6%B3%95%E8%BF%9B%E5%85%A5%E7%BD%91%E9%A1%B5luci)
